igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/client/model/Build.groovy

```diff
@@ -56,6 +58,11 @@ class Build {
      */
     @ElementList(required = false, name = "action", inline = true)
     List<TestResults> testResults
+
+    GenericBuild genericBuild(String jobName) {
```

Comment:
Yes, GenericBuild is not an interface it is meant as being the build representation of igor. This means it should not be connected to the jenkins representation of a build. It just happens to be very similar because the jenkins representation of a build is whats integrated against in the rest of the services in spinnaker.